main/helpers/asset_loader.py

```diff
@@ -1,12 +1,10 @@
-import sys, os, inspect
 import json
+from path_handler import main_dir
 
 
 class AssetLoader:
     def __init__(self):
-        self._asset_directory = (
-            os.path.dirname(os.path.realpath(inspect.getfile(self.__class__))) + "/../assets/"
-        )
+        self._asset_directory = main_dir() / "assets"
```

Comment:
You have `assets_dir()` in `path_handler` which I guess was intended for this class.
```suggestion
        self._asset_directory = assets_dir()
```

Or this attribute could be removed altogether and replaced with `assets_dir()` in `load_assets`